react/features/conference/middleware.ts

```diff
@@ -1,8 +1,11 @@
 import i18n from 'i18next';
 import { batch } from 'react-redux';
 
+// @ts-expect-error
+import { API_ID } from '../../../modules/API';
```

Comment:
This is what breaks the linter since it gets imported on mobile too. You can import it from /constants and it should work.